browser/resources/settings/brave_overrides/basic_page.js

```diff
@@ -18,6 +18,7 @@ import '../brave_wallet_page/brave_wallet_page.m.js'
 import '../brave_sync_page/brave_sync_page.js'
 import '../brave_help_tips_page/brave_help_tips_page.m.js'
 import '../brave_new_tab_page/brave_new_tab_page.m.js'
+import '../brave_search_engines_page/brave_search_engines_page.m.js'
```

Comment:
I prefer this also be sorted, but up to you.

Reply:
Done.